Makefile

```diff
@@ -7,10 +7,7 @@ lint:
 	flake8
 
 test: all
-	nosetests
-
-test-cov: all
```

Comment:
`test-cov` was one of the entrypoints we had agreed would be in all `Makefiles`. I am good with removing it (especially since we don't run coverage checks on this repo), but I thought it worth mentioning.